### DISCUSSION TITLE
Is it possible to besiege effective nodes with a large number of Socks5 agents

### DISCUSSION THREAD

**Discussion by laonadev** [Chia-Network/chia-blockchain#6712] [Support]
I have nearly 1PB plots, but get 0 XCH in the last 2 weeks.

I have checked the logs and the network connections. Everything is OK.

So I doubt whether there are attackers who use a large number of Socks5 agents or a large number of puppet nodes to siege effective nodes and prevent them from participating in screening or winning confirm information?

These attacker nodes connect to your node, which looks no different from normal nodes, except that they don't help you broadcast your winning information.

**Comment by loppefaaret** [comment]
you would only need 1 single good peer to circumvent that

**Comment by BrandtH22** [comment]
@laonadev , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
